src/explore-education-statistics-admin/src/components/editable/Comments.tsx

```diff
@@ -97,14 +97,25 @@ const Comments = ({
       });
   };
 
+  const resolveComment = (commentId: string, resolved: boolean) => {
+    const index = comments.findIndex(comment => comment.id === commentId);
+    releaseContentCommentService
```

Comment:
Why not make this function async i.e. `const resolveComment = async (.....`

Then you can do:

```
await releaseContentCommentService.updateContentSectionComment(comments[index], resolved);

const newComments = [...comments];
newComments[index] = savedComment;
onChange(blockId, newComments);
```

Reply:
Not a massive difference to be fair!